pl_bolts/models/autoencoders/basic_ae/basic_ae_module.py

```diff
@@ -138,8 +138,7 @@ def add_model_specific_args(parent_parser):
         return parser
 
 
-# todo: covert to CLI func and add test
-if __name__ == '__main__':
+def cli_main():
```

Comment:
@Borda please keep these as cli_main...
run_cli is not clear

Reply:
Ok, then let's rename all (before there were both)